src/main/java/com/example/spacecatmarket/javawebkpi/domain/Category.java

```diff
@@ -0,0 +1,13 @@
+package com.example.spacecatmarket.javawebkpi.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
```

Comment:
good, I would prefer using builder in this case, it will give you more flexibility in how you plan to create the categories https://stackoverflow.com/questions/29881135/difference-between-builder-pattern-and-constructor